include/crpropa/Source.h

```diff
@@ -606,7 +662,10 @@ class SourceDirection: public SourceFeature {
 
 /**
  @class SourceEmissionMap
- @brief Deactivate Candidate if it has zero probability in provided EmissionMap
+ @brief Deactivate Candidate if it has zero probability in provided EmissionMap. 
+
+	This feature does not change the direction of the candidate. Therefore a usefull direction feature (isotropic or directed emission)
+	must be added to the sources before. The propability of the emission map is not taken into account. 
```

Comment:
What does "the probability of the emission map is not taken into account" mean?

Reply:
it is only a filtering for p=0 or not.

Reply:
You mean that the emission map does not make a difference between emission probability p=1 or p=0.00001? 
The only parameter taken into account is (p>1) == True?

In that case I would reformulate to: "A weighting of the source emission directions is not included. Candidates are either emitted with a weight=1 for or not injected into the simulation."

Reply:
Anyway, I guess it's fine. I might add some additional comment later.